MapleServer2/Tools/InventoryController.cs

```diff
@@ -1,46 +1,153 @@
 ﻿using System;
 using MapleServer2.Types;
 using MapleServer2.Packets;
-using System.Diagnostics;
+using MaplePacketLib2.Tools;
 using MapleServer2.Servers.Game;
+using Maple2Storage.Types;
 
 public class InventoryController
 {
-	public static void Add(GameSession session, Item item)
-	{
-        // Checks if item is stackable or not
+    // Adds item
+    public static void Add(GameSession session, Item item, Boolean isNew)
+    {
+        // Checks if item is stackable or not.
         if (item.SlotMax > 1)
         {
             foreach (Item i in session.Player.Inventory.Items.Values)
             {
-                // Continue if inventory item id doesn't match or it is at max amount
+                // Checks to see if item exists in database (dictionary)
                 if (i.Id != item.Id || i.Amount >= i.SlotMax)
                 {
                     continue;
                 }
-                // Updates inventory for item amount overflow
+                // Updates inventory for item amount overflow.
```

Comment:
Please follow proper comment formatting: space after `//` no period, start with a capital letter, this is a problem throughout the PR